src/main/kotlin/no/fdk/dataservicecatalog/controller/CatalogController.kt

```diff
@@ -0,0 +1,59 @@
+package no.fdk.dataservicecatalog.controller
+
+import no.fdk.dataservicecatalog.controller.CatalogController.Companion.JSON_LD
+import no.fdk.dataservicecatalog.controller.CatalogController.Companion.N3
+import no.fdk.dataservicecatalog.controller.CatalogController.Companion.N_QUADS
+import no.fdk.dataservicecatalog.controller.CatalogController.Companion.N_TRIPLES
+import no.fdk.dataservicecatalog.controller.CatalogController.Companion.RDF_JSON
+import no.fdk.dataservicecatalog.controller.CatalogController.Companion.RDF_XML
+import no.fdk.dataservicecatalog.controller.CatalogController.Companion.TRIG
+import no.fdk.dataservicecatalog.controller.CatalogController.Companion.TRIX
+import no.fdk.dataservicecatalog.controller.CatalogController.Companion.TURTLE
+import org.apache.jena.riot.Lang
+import org.apache.jena.riot.RDFLanguages
+import org.springframework.http.HttpHeaders
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RequestHeader
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@RequestMapping("/catalogs", produces = [N3, TURTLE, RDF_XML, RDF_JSON, JSON_LD, TRIX, TRIG, N_QUADS, N_TRIPLES])
+class CatalogController {
```

Comment:
`Catalog` viser ikke hva som er forskjellem mellom denne og /catalogs-endepunktet til DataServiceController, du kan feks endre det til `RDFController` eller `PublishedController`. Ser vi har brukt RDFController i service-catalog
```suggestion
class RDFController {
```